docs/codeReview.md

```diff
@@ -0,0 +1,32 @@
+# Code review policy
+
+The goal is to help reviewers to make their intentions clear to the author and for the author to know exactly what is expected of to pass the next review round.
```

Comment:
`...what is expected of to pass...`

Should it say instead "what is expected of _them_ to pass"?